backend/todolist/src/main/java/team10/todolist/domain/Board.java

```diff
@@ -0,0 +1,82 @@
+package team10.todolist.domain;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+import team10.todolist.dto.BoardDto;
+
+import java.time.LocalDateTime;
+
+public class Board {
+
+    @Id
+    private Long id;
+    private String title;
+    private String contents;
+    private String category;
+
+    @Column(value = "date_time")
+    private LocalDateTime dateTime;
+    @Column(value = "is_deleted")
+    private int isDeleted = 0;
+
+    protected Board() {
+    }
+
+    public Board(String title, String contents, String category) {
+        this.title = title;
+        this.contents = contents;
+        this.category = category;
+        this.dateTime = LocalDateTime.now();
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public String getContents() {
+        return contents;
+    }
+
+    public String getCategory() {
+        return category;
+    }
+
+    public LocalDateTime getDateTime() {
+        return dateTime;
+    }
+
+    public int getIsDeleted() {
+        return isDeleted;
+    }
```

Comment:
```suggestion
    public int isDeleted() {
        return isDeleted;
    }
```

가급적이면 인텔리 제이 도구를 이용해서 getter 메서드를 만들어주세요.